manageiq-operator/api/v1alpha1/helpers/miq-components/postgresql_conf.go

```diff
@@ -38,8 +38,10 @@ hot_standby = on
 # ERROR REPORTING AND LOGGING
 #------------------------------------------------------------------------------
 
-log_filename = 'postgresql.log'
-log_rotation_age = 0
+log_destination = 'stderr'
+# This is used when logging to stderr:
+logging_collector = off
+
 log_min_duration_statement = 5000
 log_connections = on
 log_disconnections = on
```

Comment:
Should we drop these?  It seemed very chatty in the demo env.

Reply:
Possibly?  But, not here... I opened https://github.com/ManageIQ/manageiq-pods/issues/969 to track the autovacuum settings but we can certainly review them all to see if we need those logged.